### ISSUE TITLE
Incorrectly reporting whitespace error in media query with comment

### ISSUE THREAD

**Issue by mxstbr** [stylelint/stylelint#2018]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Stylelint incorrectly reports a "Unexpected whitespace" error when a media query contains a comment.

> Which rule, if any, is this issue related to?

`media-feature-parentheses-space-inside`

> What CSS is needed to reproduce this issue?

e.g.

```css
@media {max-width: /*comment*/) {

}
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "extends": "stylelint-config-standard"
}
```

> Which version of stylelint are you using?

`7.3.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint MyComponent.js` with `stylelint-processor-styled-components`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

It might be related to interpolations inside of `styled-components` CSS. See [this issue](https://github.com/styled-components/stylelint-processor-styled-components) in the `styled-components` processor repo for the discussion

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
MyComponent.js
2:21 ✖ Unexpected whitespace before ")"   media-feature-parentheses-space-inside
```

**Comment by davidtheclark** [comment]
Closed by #2019
